tool/embedded-protocol.js

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+const createWriteStream = require('fs').createWriteStream;
+const execSync = require('child_process').execSync;
+const https = require('https');
+
+const PROTO_NAME = 'embedded_sass.proto';
+
+/**
+ * Downloads the latest version of the Embedded Protocol into the given
+ * directory.
+ */
+function downloadProto(dir, callback) {
```

Comment:
You could probably clean this up by using https://www.npmjs.com/package/node-fetch with `async`/`await` (at the cost of adding another dev dependency).